### ISSUE TITLE
[feature request] add tooltip for terraformcost

### ISSUE THREAD

**Issue by Jamanno** [ZeroK-RTS/Zero-K#1159]
When you terraform it would be nice to see how much it costs... also when building how expensive it is.

**Comment by Sgathec** [comment]
Yes, I would also like to see the cost before and during terraforming.
